lib/policial/style_guides/ruby.rb

```diff
@@ -28,7 +28,7 @@ def default_config_file
 
       def team
         cop_classes = RuboCop::Cop::Cop.all
-        cop_classes.reject!(&:rails?) unless config['AllCops']['RunRailsCops']
+        cop_classes.reject!(&:non_rails) unless config['AllCops'].fetch('Rails', {})['Enabled']
```

Comment:
[According to the docs](https://github.com/bbatsov/rubocop#rails) `Rails` does not go under `AllCops`.